zebra-consensus/src/block/subsidy/funding_streams.rs

```diff
@@ -124,20 +124,43 @@ pub fn funding_stream_address(
 ///
 /// [7.10]: https://zips.z.cash/protocol/protocol.pdf#fundingstreams
 pub fn check_script_form(lock_script: &Script, address: Address) -> bool {
-    let mut address_hash = address
+    // TODO: Verify P2SH multisig funding stream addresses (#5577).
+    //       As of NU5, the funding streams do not use multisig addresses,
```

Comment:
This doesn't need to be addressed in this PR, but: https://github.com/ZcashFoundation/zebra/issues/5577#issuecomment-1309292394

Reply:
Thanks, I'll deal with this in ticket #5577.